lib/ui/blocful_widget.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:bloc_presentation/bloc_presentation.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+/// A widget that wraps your UI in a [BlocPresentationListener] and a [BlocConsumer] to be able to handle all events simultaniously
+///
+/// Example:
+/// /// ```dart
+/// BlocfulWidget<MyBloc, MyState, MyEvent>(
+///   onPresentationEvent: (context, event) => showSomeUIChange(),
+///   child: SomeWidget(),
+/// )
```

Comment:
Example is incorrect. Its a abstract class so it cannot be created/instantiated.